ptvsd/wrapper.py

```diff
@@ -986,7 +986,13 @@ def on_threads(self, request, args):
         # TODO: docstring
         cmd = pydevd_comm.CMD_LIST_THREADS
         _, _, resp_args = yield self.pydevd_request(cmd, '')
-        xml = self.parse_xml_response(resp_args)
+
+        try:
+            xml = self.parse_xml_response(resp_args)
+        except Exception:
```

Comment:
Is there a more specific exception type to catch for malformed XML?

Reply:
xml.sax._exceptions.SAXParseException